src/v2/view-builder/views/TerminalView.js

```diff
@@ -12,24 +12,35 @@ const OPERATION_CANCELED_BY_USER_KEY = 'idx.operation.cancelled.by.user';
 const DEVICE_ACTIVATED = 'idx.device.activated';
 const DEVICE_NOT_ACTIVATED_CONSENT_DENIED = 'idx.device.not.activated.consent.denied';
 const DEVICE_NOT_ACTIVATED_INTERNAL_ERROR = 'idx.device.not.activated.internal.error';
+const FLOW_CONTINUE_IN_NEW_TAB = 'idx.transferred.to.new.tab';
+const EMAIL_LINK_OUT_OF_DATE = 'idx.return.stale';
+const EMAIL_LINK_CANT_BE_PROCESSED = 'idx.return.error';
+const EMAIL_FINISH_SIGN_IN = 'idx.email.verification.required';
```

Comment:
`nit:` Can we rename this to something in the lines of `EMAIL_VERIFICATION_REQUIRED`? The current wording is a bit confusing.